ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -528,10 +536,80 @@ protected void initTreeTable() {
     doInit();
   }
 
+  private static Comparator<Task> beginComparatorAsc = new Comparator<Task>() {
```

Comment:
I don't understand how I can get TaskDefaultColumn from col index in GPTreeTableBase, could you help my with it&